src/nupic/algorithms/SpatialPooler.cpp

```diff
@@ -772,7 +773,7 @@ void SpatialPooler::updatePermanencesForColumn_(vector<Real>& perm,
   }
 
   numConnected = 0;
-  for (UInt i = 0; i < perm.size(); ++i)
+  for (UInt i = 0; i < perm.size(); i++)
```

Comment:
thanks @scottpurdy , I've missed to realize it's the same in a for-loop. removing the change.